src/Components/Server/src/Prerendering/ComponentPrerrenderingContext.cs

```diff
@@ -1,10 +1,10 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
 using Microsoft.AspNetCore.Http;
 
-namespace Microsoft.AspNetCore.Components.Server
+namespace Microsoft.AspNetCore.Components.Server.Prerendering
```

Comment:
Do we really need another namespace?

Reply:
I just didn't want to pollute the top namespace for a type that the user is not likely to ever reference directly, but will move it to it if you are ok with it.

Reply:
got rid of the ns